quiz.js

```diff
@@ -1,25 +1,38 @@
 var treeObject = {};
-document.getElementById("growTreeBtn").addEventListener("click", tree);
+
+/*****************EVENT LISTENERS**************************/
+
+document.getElementById("growTreeBtn").addEventListener("click", clickListener);
 document.getElementById("character").addEventListener("keydown", enterListener);
 document.getElementById("treeHeight").addEventListener("keydown", enterListener);
 
 function enterListener(key) {
 	if (13 == key.keyCode) {
-		tree()
+		tree(treeObject);
 	}
 }
 
+function clickListener() {
+	tree(treeObject);
+}
+
+/*****************TREE BUILD**************************/
+
+function tree(obj) {
+	obj.hgt = Math.abs(document.getElementById('treeHeight').value);
+	obj.char = document.getElementById('character').value;
+	if (obj.hgt == 0) {
```

Comment:
Missing a bock of code here. I'm guessing you're dealing with the edge case of a height that's 0 or a negative number? If so, this is a good place to throw an error or some kind to let your user know they need to add a correct input.